helm/fiftyone-teams-app/README.md

```diff
@@ -813,49 +814,99 @@ or modify your existing configuration to migrate to a new Auth0 Tenant.
       --from-file=license=./your-license-file
     ```
 
-1. [Upgrade to FiftyOne Teams version 2.0.0](#deploying-fiftyone-teams)
-1. Upgrade FiftyOne Teams SDK users to FiftyOne Teams version 2.0.0
+1. [Upgrade to FiftyOne Teams version 2.0.1](#deploying-fiftyone-teams)
+1. Upgrade FiftyOne Teams SDK users to FiftyOne Teams version 2.0.1
     - Login to the FiftyOne Teams UI
     - To obtain the CLI command to install the FiftyOne SDK associated with
       your FiftyOne Teams version, navigate to `Account > Install FiftyOne`
 1. Upgrade all the datasets
 
-    > **NOTE** Any FiftyOne SDK less than 2.0.0 will lose connectivity after
+    > **NOTE** Any FiftyOne SDK less than 2.0.1 will lose connectivity after
     > this point.
-    > Upgrading all SDKs to `fiftyone==2.0.0` is recommended before migrating
+    > Upgrading all SDKs to `fiftyone==2.0.1` is recommended before migrating
         > your database.
 
     ```shell
     FIFTYONE_DATABASE_ADMIN=true fiftyone migrate --all
     ```
 
-1. Validate that all datasets are now at version 0.25.0
+1. Validate that all datasets are now at version 0.25.1
 
     ```shell
     fiftyone migrate --info
     ```
 
+### From FiftyOne Teams Version 2.0.0
+
+1. [Upgrade to FiftyOne Teams version 2.0.1](#deploying-fiftyone-teams)
+1. Voxel51 recommends upgrading all FiftyOne Teams SDK users to FiftyOne Teams
+   version 2.0.1, but it is not required
+    - Login to the FiftyOne Teams UI
+    - To obtain the CLI command to install the FiftyOne SDK associated with
+      your FiftyOne Teams version, navigate to `Account > Install FiftyOne`
+1. Voxel51 recommends that you upgrade all your datasets, but it is not
+   required.  Users using the FiftyOne Teams 2.0.0 SDK will continue to operate
+   uninterrupted during, and after, this migration
+
+   ```shell
+   FIFTYONE_DATABASE_ADMIN=true fiftyone migrate --all
+   ```
+
+1. To ensure that all datasets are now at version 0.25.1, run
+
+   ```shell
+   fiftyone migrate --info
+   ```
+
 ## Deploying FiftyOne Teams
 
 A minimal example `values.yaml` may be found
 [here](https://github.com/voxel51/fiftyone-teams-app-deploy/blob/main/helm/values.yaml).
 
 1. Edit the `values.yaml` file
 1. Deploy FiftyOne Teams with `helm install`
-    1. For a new installation, run
-
-        ```shell
-        helm repo add voxel51 https://helm.fiftyone.ai
-        helm repo update voxel51
-        helm install fiftyone-teams-app voxel51/fiftyone-teams-app -f ./values.yaml
-        ```
-
-    1. To upgrade an existing helm installation, run
-
-        ```shell
-        helm repo update voxel51
-        helm upgrade fiftyone-teams-app voxel51/fiftyone-teams-app -f ./values.yaml
-        ```
+    1. For a new installation
```

Comment:
The namespace must be created before creating resources in the namespace. For example:

```
$ kubectl apply -f secret-legacy-license.yaml --namespace does-not-exist --context minikube
Error from server (NotFound): error when creating "secret-legacy-license.yaml": namespaces "does-not-exist" not found
```